crates/uv/src/commands/mod.rs

```diff
@@ -82,3 +90,30 @@ pub(crate) enum VersionFormat {
     Text,
     Json,
 }
+
+pub(super) async fn compile_and_report(
```

Comment:
Nit: let's just call this `compile`? Feels more consistent with other commands. Can you add rustdoc too?